src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/Extensions/RazorCodeDocumentExtensions.cs

```diff
@@ -10,7 +10,7 @@ internal static class RazorCodeDocumentExtensions
 {
     private static readonly object s_unsupportedKey = new();
 
-    public static bool IsUnsupported(this RazorCodeDocument document)
+    internal static bool IsUnsupported(this RazorCodeDocument document)
```

Comment:
Can this be put back to public, given the class is internal?

Reply:
oh, is this because RazorCodeDocument is internal now?

Reply:
> oh, is this because RazorCodeDocument is internal now?

Correct.